scripts/orchestra-manager.py

```
@@ -1325,6 +1335,10 @@ def main():
         else:
             facet_parser.print_help()
             sys.exit(1)
+    elif args.command == "meta":
+        meta_script = orchestra_dir / "packages" / "meta-harness" / "scripts" / "meta_harness.py"
+        result = subprocess.run([sys.executable, str(meta_script)] + args.meta_args)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)